lib/jekyll/converter.rb

```diff
@@ -8,7 +8,9 @@ class Converter < Plugin
     #
     # Returns the String prefix.
     def self.highlighter_prefix(highlighter_prefix = nil)
-      @highlighter_prefix = highlighter_prefix if highlighter_prefix
+      if !defined?(@highlighter_prefix) || !highlighter_prefix.nil?
```

Comment:
This changes the behaviour slightly. Before: If `highlighter_prefix` is truthy, set its value to `@highlighter_prefix`. After: if `highlighter_prefix` is not nil or `@highlighter_prefix` isn't defined, then it sets `@highlighter_prefix` to `highlighter_prefix`. 

Why add the `!defined?` check?